### DISCUSSION TITLE
Custom SCSS

### DISCUSSION THREAD

**Discussion by gbeifuss** [CaiJimmy/hugo-theme-stack#303] [Q&A] [answered]
Hi! I'm trying to tweak the theme so that code blocks use a slightly smaller font size.
I thought I could do this by adding a definition for PRE into hugo-theme-stack\assets\custom.scss:
```
pre {
  display: block;
  font-family: monospace;
  white-space: pre;
  margin: 1em 0;
  font-size: 10px;
} 
```
Is this right, or even the right spot?

**Comment by CaiJimmy** [comment, marked as answer]
Use `.article-content pre` instead of `pre` only, to be more specific. Otherwise, it might fail overwriting the default style.

**Reply by gbeifuss** [reply]
Thanks for the reply!

For future searchers, I ended up using:
.article-content pre {
  display: block;
  font-family: monospace;
  white-space: pre;
  margin: 1em 0;
  font-size: 0.85em;
}
